include/library/spdm_lib_config.h

```diff
@@ -235,6 +235,23 @@
 #define LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP 1
 #endif
 
+/* When LIBSPDM_ENABLE_CAPABILITY_RSP_IF_READY is 0 then
+ *      - For a Requester, if the Responder sends a ResponseNotReady ERROR response then the error
+ *        is immediately returned to the Integrator. The Requester cannot send a RESPOND_IF_READY
+ *        request.
+ *      - For a Responder, it cannot send a RESPOND_IF_READY ERROR response and does not support
+ *        RESPOND_IF_READY.
+ * When LIBSPDM_ENABLE_CAPABILITY_RSP_IF_READY is 1 then
+ *      - For a Requester, if the Responder sends a ResponseNotReady ERROR response then libspdm
+ *        waits an amount of time, as specified by the RDTExponent parameter, before sending
+ *        RESPOND_IF_READY.
+ *      - For a Responder, if its response state is NOT_READY then it will send a ResponseNotReady
+ *        ERROR response to the Requester, and will accept a subsequent RESPOND_IF_READY request.
+ */
+#ifndef LIBSPDM_ENABLE_CAPABILITY_RSP_IF_READY
+#define LIBSPDM_ENABLE_CAPABILITY_RSP_IF_READY 1
```

Comment:
Prefer to call it `LIBSPDM_RSP_IF_READY_SUPPORT`

Reply:
It is done.